pages/blog.html

```diff
@@ -0,0 +1,123 @@
+<!DOCTYPE html>
+<html lang="en">
+   <head>
+      <meta charset="UTF-8">
+      <meta http-equiv="X-UA-Compatible" content="IE=edge">
+      <meta name="viewport" content="width=device-width, initial-scale=1.0">
+      <title>A yr of Code Blog | Read me </title>
+      <link rel="stylesheet" href="styles/style.css">
+      <link rel="stylesheet" media="screen and (max-width: 768px)" href="css/mobile.css">
+      <link rel="shortcut icon" type="image/x-icon" href="favicon.ico">
+      <link rel="preconnect" href="https://fonts.googleapis.com">
+      <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+      <link href="https://fonts.googleapis.com/css2?family=Lato&display=swap" rel="stylesheet">
+   </head>
+   <body class="blogmain">
+      <div class = "container">
+         <!-- Navbar and Page Menu -->
+         <div class="header box">
+            <div class="logo">
+               <a class="logo" href="bloghome.html">
+               <img src="images/bloglogo.png.png" alt="Homepage">
+               </a>
+               <h1>
+                  A year of Code Blog
+               </h1>
+               <ul class="listnav">
+                  <li><a href="#">About</a></li>
+               </ul>
+               <ul class="listnav">
+                  <li><a href="blog.html"> Most Current Blog Post</a></li>
+               </ul>
+            </div>
+         </div>
+      </div>
+      <div class="blogcontent">
+         <h2> BLOG TITLE Heading </h2>
+         <h5> Title description , DATE </h5>

```

Comment:
Jumping from `h2` to `h5` makes me think these are being used primarily for their intrinsic styles. Prefer to use heading tags in order, and use style rules to adjust their look as desired.